NodeJS/src/AgentSkill.ts

```diff
@@ -0,0 +1,47 @@
+const args = require('minimist')(process.argv.slice(2));
+export default args
+
+const ENDPOINT_KEY = 'endpoint';
+const ADMIN_USERNAME_KEY = 'admin_username';
+const ADMIN_PASSWORD_KEY = 'admin_password';
+const REPLACE_REGEX = /'/g;
```

Comment:
what is being replaced here?

Reply:
--endpoint='https://integration.bpo.avaya.com'
--endpoint=https://integration.bpo.avaya.com 
Both will work.. replacing single inverted.